gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/longrunning/stub/OperationsStubSettings.java

```diff
@@ -293,13 +292,13 @@ public static class Builder extends StubSettings.Builder<OperationsStubSettings,
       RetrySettings settings = null;
       settings =
           RetrySettings.newBuilder()
-              .setInitialRetryDelay(Duration.ofMillis(500L))
+              .setInitialRetryDelayDuration(java.time.Duration.ofMillis(500L))
               .setRetryDelayMultiplier(2.0)
-              .setMaxRetryDelay(Duration.ofMillis(10000L))
-              .setInitialRpcTimeout(Duration.ofMillis(10000L))
+              .setMaxRetryDelayDuration(java.time.Duration.ofMillis(10000L))
+              .setInitialRpcTimeoutDuration(java.time.Duration.ofMillis(10000L))
               .setRpcTimeoutMultiplier(1.0)
-              .setMaxRpcTimeout(Duration.ofMillis(10000L))
-              .setTotalTimeout(Duration.ofMillis(10000L))
+              .setMaxRpcTimeoutDuration(java.time.Duration.ofMillis(10000L))
+              .setTotalTimeoutDuration(java.time.Duration.ofMillis(10000L))
               .build();
```

Comment:
So I believe this part is actually a weird state. This file in the Operations client is GAPIC generated file, but HttpJsons Operations Client actually can't be regenerated (as of now).

I think we can apply these manual changes here, but this is a reminder that we probably also need to make the changes inside the generator as well (once these gax changes go in).

Reply:
Added an issue to track this: https://github.com/googleapis/sdk-platform-java/issues/2859